### ISSUE TITLE
Only the checkmark appears when branch has 0 conflicts with the "base" branch

### ISSUE THREAD

**Issue by agisilaos** [desktop/desktop#5689]
## Description

Recently I had to go through a merge in one of our branches over at Iris. The branch had 0 conflicts with the base branch and it was up-to-date. Then I saw at the bottom of the app (below the merge button) only a checkmark. So it got me thinking. It felt weird to me to get a message there when conflicts exist and to just present a checkmark when the branch has 0 conflicts. 🤔 

## Version

* GitHub Desktop: 1.4.1-beta0

* Operating system: macOS High Sierra v10.13.6 

## Steps to Reproduce
1. Make sure you created a branch from your default branch
2. Tap in the History tab
3. Tap in the field
4. Write the name of your default branch (e.g master)
5. Select your default branch from the list
6. Observe the area of the app below the commit button

### Expected Behavior
I expect from the app to present a message below the checkmark or not present the checkmark at all. 🤔 Still thinking about that.

### Actual Behavior
It only presents the checkmark. 

-- 

P.s Title of the issue is too broad probably. I'm open to suggestions for a better title for the issue. 
## Additional Information

![screen shot 2018-09-10 at 20 24 02](https://user-images.githubusercontent.com/10944108/45818449-74a2de00-bcea-11e8-9849-05c23f160a75.png)

**Comment by shiftkey** [comment]
@agisilaos thanks for the report! I had a go at reproducing this but my new branch `making-new-things` (0 ahead and 0 behind `master`) didn't display anything below the commit button:

<img width="532" alt="screen shot 2018-09-20 at 9 49 51 am" src="https://user-images.githubusercontent.com/359239/45819432-ad2ac380-bcba-11e8-9407-99f143ee45b3.png">

<img width="277" src="https://user-images.githubusercontent.com/359239/45819401-9be1b700-bcba-11e8-8db2-987f1d22e23d.png">

Could there be some detail about the repro steps I've overlooked?

**Comment by agisilaos** [comment]
Oh I just checked that the screenshot I provided is from a while ago and as I can see in #5581 you already fixed the issue. @shiftkey

**Comment by shiftkey** [comment]
@agisilaos it still seems weird that you're able to reproduce it on `1.4.1-beta0` (which should have that fix). Let me know if you're able to stumble upon it again.

**Comment by agisilaos** [comment]
@shiftkey Hmm, I'm pretty sure that when I stumbled upon this issue my Github Desktop version wasn't `1.4.1-beta0`.

When I stumbled upon this issue I took a screenshot but I didn't remember in which version of GH desktop happened. So today when I opened the issue I added `1.4.1-beta0` as the version of Github desktop since I thought it will be still reproducible. It turns out it wasn't. You def fix it with #5581.
